### ISSUE TITLE
subview should be appended so that it follows notify title/body

### ISSUE THREAD

**Issue by chris-steele** [adaptlearning/adapt-contrib-core#158]
At present the subview is _prepended_ which means the read order is subview content > title > body for screen readers

See https://github.com/adaptlearning/adapt-contrib-core/blob/master/js/views/notifyPopupView.js#L188
